ada/language/ast.py

```diff
@@ -3682,19 +3682,25 @@ def primitive_type_accessor():
         return T.PrimTypeAccessor.new(prim_type=Entity)
 
     @langkit_property(memoized=True)
+    def compute_primitives_env(include_self=(Bool, True)):
```

Comment:
FIY: stopping there, you already have lots of work to do with the previous comments. However: this seems to be severely lacking comments !

Reply:
Okay I added a comment but note that I didn't change any major logic here, this new property simply allows me to factor our a common piece of code (the part that does `group_env` with the explicit `with_md` argument) in both primitives_env and parent_primitives_env. (It was actually missing from `primitives_env` and so this refactoring allows fixing that conveniently)